apps/codeforafrica/src/lib/data/blockify/ourTeam.js

```diff
@@ -1,23 +1,32 @@
 import { getMembers } from "@/codeforafrica/lib/data/utils/members";
+import { sortTags } from "@/codeforafrica/lib/data/utils/tags";
 
+function getTeamTags(docs) {
+  const tags = sortTags(docs.map((item) => item.team).filter(Boolean));
+  return [{ name: "All", slug: "All" }, ...tags];
+}
+
+function getStringTags(field, docs) {
+  const tags = sortTags(
+    docs
+      .map((item) =>
+        item[field] ? { name: item[field], slug: item[field] } : null,
+      )
+      .filter(Boolean),
+  );
+  return [{ name: "All", slug: "All" }, ...tags];
+}
```

Comment:
What does `String` mean in getStringTags?

Reply:
We want to get country tags for example, they exist as strings, not objects.

Reply:
1. Is there any other field besides country that this method is handling? If not, we should call getCountryTags
2. We were supposed to be using this [file](https://github.com/CodeForAfrica/ui/blob/main/apps/codeforafrica/src/lib/data/json/countries.js) I think to have country `name` -> `iso alpha3 code` as object. Not sure why we didn't do that.


I think we're using something like that in charterAFRICA (check somewhere around Map/Focal countries) or searching online should also give you an idea of what I'm [talking](https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes) [about](https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes). If it's not too late, we should switch to that format.

Reply:
2. We used this file as options in cms but it is stored as a string(value). We also do not want all countries listed for filtering. we only need those that exist in the team collection. @kilemensi

Reply:
Yes @koechkevin, I full understand that. What I mean is:

1. We should switch the file used in the CMS to the one that has name and iso code instead of just name for both label and value.
2. Yes, we don't want to show all countries but once we get the string (which will be a code this time), we'll use this file to format it into object i.e {.name: "Name of the country", slug: iso code } as required by tag handling function.